lit.ts

```diff
@@ -217,6 +236,61 @@ export async function storeConditionWithSigner(
 	return tx;
 }
 
+export async function mintPKPV3({
+	keyType,
+	permittedAuthMethodTypes,
+	permittedAuthMethodIds,
+	permittedAuthMethodPubkeys,
+	permittedAuthMethodScopes,
+	addPkpEthAddressAsPermittedAddress,
+	pkpEthAddressScopes,
+	sendPkpToItself,
+	burnPkp,
+}: {
+	keyType: string;
+	permittedAuthMethodTypes: string[];
+	permittedAuthMethodIds: string[];
+	permittedAuthMethodPubkeys: string[];
+	permittedAuthMethodScopes: string[][];
+	addPkpEthAddressAsPermittedAddress: boolean;
+	pkpEthAddressScopes: string[][];
+	sendPkpToItself: boolean;
+	burnPkp: boolean;
+}): Promise<ethers.Transaction> {
+	console.log(
+		"In mintPKPV3",
+		keyType,
+		permittedAuthMethodTypes,
+		permittedAuthMethodIds,
+		permittedAuthMethodPubkeys,
+		permittedAuthMethodScopes,
+		addPkpEthAddressAsPermittedAddress,
+		sendPkpToItself,
+	);
+
+	console.log('config.network:', config.network);
+
+	const pkpHelper = await getPkpHelperV2Contract();
+	const pkpNft = await getPkpNftContract();
+
+	// first get mint cost
+	const mintCost = await pkpNft.mintCost();
+	const tx = await pkpHelper.mintNextAndAddAuthMethods(
```

Comment:
interesting that this implementation is not using the queuing system which will guarantee ordering `nonce` any reason for this not to be used?